contracts/Permissions.sol

```diff
@@ -19,23 +19,22 @@
     along with SKALE Allocator.  If not, see <https://www.gnu.org/licenses/>.
 */
 
-pragma solidity 0.6.10;
+pragma solidity 0.8.11;
 
-import "@openzeppelin/contracts-upgradeable/math/SafeMathUpgradeable.sol";
-import "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
-import "./thirdparty/openzeppelin/InitializableWithGap.sol";
+import "@skalenetwork/skale-manager-interfaces/IContractManager.sol";
+import "@skalenetwork/skale-manager-interfaces/IPermissions.sol";
 
-import "./interfaces/IContractManager.sol";
+import "./thirdparty/AccessControlUpgradeableLegacy.sol";
 
 
 /**
  * @title Permissions - connected module for Upgradeable approach, knows ContractManager
  * @author Artem Payvin
  */
-contract Permissions is InitializableWithGap, AccessControlUpgradeable {
-    using SafeMathUpgradeable for uint;
+contract Permissions is AccessControlUpgradeableLegacy, IPermissions {
     using AddressUpgradeable for address;
 
+    // slither-disable-next-line uninitialized-state
```

Comment:
Delete this value somewhere in the code. It frees the state and remove unuzed-variable warning.